docs/sql_reference/src/asciidoc/_chapters/sql_functions_and_expressions.adoc

```diff
@@ -31,20 +31,27 @@ and expressions that you can use in {project-name} SQL statements. The
 functions and expressions are categorized according to their
 functionality.
 
-[[standard_normalization]]
-== Standard Normalization
+[[categories]]
+== Categories
 
-For datetime functions, the definition of standard normalization is: If
-the ending day of the resulting date is invalid, the day will be rounded
-DOWN to the last day of the result month.
+Use these types of functions within an SQL value expression:
 
-== Aggregate (Set) Functions
+** Aggregate (Set) Functions
```

Comment:
Do you want to include LOB Functions and Encryption Functions in this list?

Reply:
@DaveBirdsall Thanks, I've added 😁